Tetris.js

```diff
@@ -45,39 +45,37 @@ class Gameboard {
     startGame() {
         const currentSquarePos = { x: 4, y: 0 };
 
-        const s1 = new Square(
+        let s1 = new Square(
             this.gridToPixel(currentSquarePos),
             this.c, this.baseUnitSideLength, "lightblue", this.bgColor);
 
         s1.display();
 
-        const rows = this.rows;
-        const grid = this.grid;
+        const gInterval = setInterval(() => {
+            console.log(this.grid[currentSquarePos.y][currentSquarePos.x]);
+            if (currentSquarePos.y < this.rows - 1 &&
+                this.grid[(currentSquarePos.y) + 1][currentSquarePos.x] !== true) {
```

Comment:
better to use something like this.isCellOccupied(x, y)

Reply:
And accordingly, we might have this.occupyCell(x, y) to mark a cell as occupied.

Reply:
done